stats/jpgraph_files/round_backlog.php

```diff
@@ -63,46 +57,34 @@
 //              Try to keep the translation the same length as the English text.
 $x_title = _("Help is most needed in the red rounds");
 
-// Why not scale from zero to max value?
-$graph->SetScale("textlin", 0, max($datay) * 1.1);
-$graph->graph_theme = null;
-$graph->img->SetAntiAliasing();
-
-// Set background to white
-$graph->SetMarginColor('white');
-
-// Add a drop shadow
-$graph->SetShadow();
-
-// Adjust the margin a bit to make more room for titles
-// left, right, top, bottom
-$graph->img->SetMargin(50, 20, 30, 60);
-
-// Set title
-$graph->title->Set($title);
-$graph->title->SetFont($jpgraph_FF, FS_BOLD, 10);
-
-// Set X axis
-$graph->xaxis->SetTickLabels($datax);
-$graph->xaxis->title->Set($x_title);
-$graph->xaxis->title->SetFont($jpgraph_FF, $jpgraph_FS);
-$graph->xaxis->title->SetMargin(10);
-$graph->xaxis->HideTicks();
-
-// Set Y axis
-$graph->yaxis->HideTicks();
-
-// Create a bar plot
-$plot = new BarPlot($datay);
-$plot->SetFillColor($barColors);
-$graph->Add($plot);
-
-// Add display of values at top of bars
-$plot->value->Show();
-$plot->value->SetColor("black");
-$plot->value->SetFont($jpgraph_FF, $jpgraph_FS, 8);
-$plot->value->SetAngle(0);
-$plot->value->SetFormat("%d");
-
-// Display the graph
-$graph->Stroke();
+// If this is a new system there won't be any stats so don't divide by zero
+if ($stats_total == 0) {
+  $title = _("No pages found.");
```

Comment:
This should have 4 spaces -- did the formatter not catch this? Here and below in the `slim_header()` call.

Reply:
It can't catch it if I don't run it :/. I thought the code formatter checked in the checks on github, but not for this branch since it forked before all that work was finished.